app/src/main/java/fr/free/nrw/commons/feedback/Feedback.java

```diff
@@ -0,0 +1,90 @@
+package fr.free.nrw.commons.feedback;
+
+public class Feedback {
```

Comment:
Our policy is that each added class, method, property should have some javadoc. Thanks! :-)